sdk/storage/Azure.Storage.DataMovement.Files.Shares/tests/ShareDirectoryStorageResourceContainerTests.cs

```diff
@@ -87,7 +87,8 @@ public void GetCorrectStorageResourceItem()
 
             // Get the subpath resource item
             StorageResourceItem resourceItem = resourceContainer.GetStorageResourceReferenceInternal(
-                string.Join("/", pathSegments));
+                string.Join("/", pathSegments),
+                default);
```

Comment:
nit: Should we pass something in here for completeness? It would never ideally be `null`. (Doesn't really matter since it's a test of something that doesn't use it)

Reply:
Ideally not, I left it as default because in this particular case, we don't do anything with the resourceId, so it wouldn't do anything. I will change it to "ShareFile" though.

Reply:
Done